### DISCUSSION TITLE
using-pathbase-with-dotnet-6-webapplicationbuilder/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#8] [Announcements]
# using-pathbase-with-dotnet-6-webapplicationbuilder/

In this post I describe the difficulties of adding calls to UsePathBase with .NET 6 WebApplication programs, and describe two approaches to work around it

https://andrewlock.net/using-pathbase-with-dotnet-6-webapplicationbuilder/

**Comment by Storm-BE** [comment]
Thanks for this article, Andrew! Great explanation, and it solved my issue ;)

[a comment by an automated account is not shown]

**Comment by jitangupta** [comment]
Hey Andrew, thanks for this articles. Good one. :)
